### PR TITLE
Lib 227 dev site nav

### PR DESCRIPTION
**Pull request by maxwellgordon**
Code Review PR for [LIB-227](https://dfuzrdev.atlassian.net/browse/LIB-227)

### CODE REVIEW

File: website/static/js/mobile_nav.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  let isSubNavClick = false;
+  document.addEventListener('DOMContentLoaded', function() {
+    addClickHandlersToMainNavTriggers();
+    addClickHandlersToSubNavTriggers();
+  });
+
+  function addClickHandlersToMainNavTriggers() {
+    const mainNavToggles = document.querySelectorAll('.main-nav .mobile.main-nav-trigger');
+    const mainNav = document.querySelector('.main-nav .navigationWrapper.navigationSlider');
+    window.addEventListener('click', (event) => {
```
Comment (jjwill) [inline website/static/js/mobile_nav.js]: I don't think arrow functions work with IE. Not positive though. I needed to remove all arrow functions and string templates from `segment.js`. We can wait for Beth to verify.

Comment (oriooctopus) [reply website/static/js/mobile_nav.js]: Don't we have babel? Babel is supposed to handle compatibility issues with ES6

Comment (maxwellgordon) [reply website/static/js/mobile_nav.js]: This is a static file. It is served as is. So babel does not compile it.

**Comment by oriooctopus** [comment]
GGGGone

**Pull request closed without merging**